agent/command/s3_get.go

```diff
@@ -78,6 +78,11 @@ type s3get struct {
 	// RoleARN is an ARN that should be assumed to make the S3 request.
 	RoleARN string `mapstructure:"role_arn" plugin:"expand"`
 
+	// RequireChecksumSha256 will verify the sha256 checksum inside
+	// the file's metadata with the given one. The checksum is Base64
+	// encoded.
+	RequireChecksumSha256 string `mapstructure:"require_checksum_sha256" plugin:"expand"`
```

Comment:
The [test yaml](https://github.com/evergreen-ci/commit-queue-sandbox/blob/bdee0096aee867c278fd85e0761c20e3a5d4c9df/s3-yaml.yaml#L238) simply uses checksum_sha256 for both s3.get and s3.put. I'm guessing you changed it after.

Reply:
I agree that this is an improvement. thank you!